src/rpc/server/scala/api/src/main/scala/com/ctc/omega_edit/SessionImpl.scala

```diff
@@ -91,10 +91,7 @@ private[omega_edit] class SessionImpl(p: Pointer, i: FFI) extends Session {
     *   https://github.com/ctc-oss/omega-edit/wiki#undo
     */
   def undoLast(): Result =
-    i.omega_edit_undo_last_change(p) match {
-      case 0 => Change.Fail
-      case v => Changed(v)
-    }
+    Edit(i.omega_edit_undo_last_change(p))
```

Comment:
@Shanedell, just wanted to make sure you understand why I'm stripping out the error handling in the RPC server.  What was going on is that if the edit failed due to changes being paused or some other error, the RPC server would throw an unserializable (resulting in a message not being sent to the client) exception.  Instead what we need to do is pass failed edits through to the client and let _it_ decide how to handle them.  The RPC server really just needs to pass things through without doing much else.  It could be enhanced later to maybe log things that are atypical, but it shouldn't try to do any intervening itself or throw anything on errors (what is going to catch them anyway?).